r1cs-std/src/fields/mod.rs

```diff
@@ -430,13 +453,23 @@ pub(crate) mod tests {
 
         let a_inv = a.inverse(cs.ns(|| "a_inv")).unwrap();
         a_inv
-            .mul_equals(cs.ns(|| "check_equals"), &a, &one)
+            .mul_equals(cs.ns(|| "check a_inv * a = 1"), &a, &one)
```

Comment:
Went ahead and fixed this namespace in the inverse test to be descriptive, so as to avoid namespace collisions.